source/about/people/institutions_advisory_council.md

```diff
@@ -27,13 +27,22 @@ Maureen P. Walsh is the Scholarly Sharing Strategist and an Associate Professor
 
 **Tracey Clarke (Member, 2023-2026),** representing Jisc, United Kingdom
 
+Tracey was formerly the Associate Director (Digital) at the University of Sheffield (UK) Library having held a number of posts with digital, open access, publishing, research support, collection development, copyright and faculty engagement responsibilities for over 25 years. Tracey has led for the university with regard to open access, research data management and digital preservation. Since 2019 Tracey has been active as a specialist consultant with an international outlook, retaining a strong interest in research ecosystems and innovation in the scholarly communications paradigm.
+
+
 **Robert Harington (Member, 2023-2026),** representing the Society Group
 
 Robert is Associate Executive Director, Publishing at the American Mathematical Society (AMS).
 Robert is a member of the AMS leadership team, and has the overall responsibility for publishing at the AMS, including books, journals and electronic products. Robert is a "Chef" for the Society for Scholarly Publishing (SSP) blog, The Scholarly Kitchen. Robert serves on the Board of Directors of CHOR Inc., (CHORUS). Robert has worked in non-profit and commercial settings, internationally, with particular experience across the United States, Europe, and Asia (focus on China and Japan). Robert holds a doctorate in biochemistry from the University of Oxford, and a first-class honors degree in chemistry from the University of London.
 
 **Martin Kohler (Member, 2023-2026),** representing Consortium arXiv-DH and HGF, Germany
 
+Martin heads the DESY and European XFEL libraries including DESY’s institutional repository, open access and publishing house. He holds a doctorate in physics from the university of Hamburg.
```

Comment:
DESY and XFEL should be both spelled out once. 
Deutsches Elektronen-Synchrotron
X-Ray Free-Electron Laser

Reply:
edited